static/js/clientHooks.js

```diff
@@ -159,16 +159,18 @@ exports.postToolbarInit = function (hook_name, args) {
 }
 
 exports.postAceInit = function(hook, context) {
-  $.getScript("http://ajax.googleapis.com/ajax/libs/jqueryui/1.11.1/jquery-ui.min.js", function(script, textStatus, jqXHR) {
-    autocomplete.postAceInit(hook, context);
-    toolbarAlert.postAceInit(hook, context);
-  });
+  require('ep_mypads/static/js/jquery-ui');
```

Comment:
Call the jquery-ui require inside autocomplete and toolbaralert respectively.  So that its obvious that clienthooks is dependent on autocomplete and toolbarAlert, which both just so happen to require jquery-ui (currently its not clear what modules use jquery-ui).  Its ok if the require is called twice, (there's special handling that reuses the importing efficiently).